core/feed_protocol/opds.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING, Any, Optional
+
+from core.feed_protocol.base import FeedProtocol
+from core.feed_protocol.serializer.opds import OPDS1Serializer
+from core.feed_protocol.types import FeedData, WorkEntry
+from core.util.flask_util import OPDSEntryResponse, OPDSFeedResponse
+
+if TYPE_CHECKING:
+    pass
+
+
+class OPDSFeedProtocol(FeedProtocol):
+    def __init__(
+        self, title: str, url: str, precomposed_entries: Optional[Any] = None
+    ) -> None:
+        self.url = url
+        self.title = title
+        self._precomposed_entries = precomposed_entries
+        self._feed = FeedData()
+        self._serializer = OPDS1Serializer()
+
+    def generate_feed(self) -> None:
+        pass
```

Comment:
Since its abstract in the interface, and its not implemented here, it seems like the `BaseOPDSFeed` class should be an ABC, and this should just not be implemented. Does that make sense? Is `BaseOPDSFeed` ever expected to be instantiated?

Reply:
You're right, I thought I was using this during in the tests, but I am not.